src/main/kotlin/eu/europa/ec/eudi/pidissuer/adapter/out/pid/EncodePidInSdJwtVc.kt

```diff
@@ -134,55 +134,56 @@ private fun selectivelyDisclosed(
         //
         // Always disclosed claims
         //
-        iss(credentialIssuerId.externalForm)
-        iat(iat.toInstant().epochSecond)
-        nbf?.let { nbf(it.epochSecond) }
-        exp(exp.epochSecond)
+        claim("iss", credentialIssuerId.externalForm)
+        claim("iat", iat.toInstant().epochSecond)
+        nbf?.let { claim("nbf", it.epochSecond) }
+        claim("exp", exp.epochSecond)
         cnf(holderPubKey)
-        plain("vct", vct.value)
+        claim("vct", vct.value)
 
         //
         // Selectively Disclosed claims
         //
-        sd(OidcFamilyName.name, pid.familyName.value)
-        sd(OidcGivenName.name, pid.givenName.value)
-        sd(OidcBirthDate.name, pid.birthDate.toString())
-        structured(Attributes.AgeEqualOrOver.name) {
-            pid.ageOver18?.let { sd(Attributes.AgeOver18.name, it) }
+        sdClaim(OidcFamilyName.name, pid.familyName.value)
+        sdClaim(OidcGivenName.name, pid.givenName.value)
+        sdClaim(OidcBirthDate.name, pid.birthDate.toString())
+        objClaim(Attributes.AgeEqualOrOver.name) {
+            pid.ageOver18?.let { sdClaim(Attributes.AgeOver18.name, it) }
         }
-        pid.ageInYears?.let { sd(Attributes.AgeInYears.name, it.toInt()) }
-        pid.ageBirthYear?.let { sd(Attributes.AgeBirthYear.name, it.value.toString()) }
-        pid.familyNameBirth?.let { sd(OidcAssuranceBirthFamilyName.name, it.value) }
-        pid.givenNameBirth?.let { sd(OidcAssuranceBirthGivenName.name, it.value) }
+        pid.ageInYears?.let { sdClaim(Attributes.AgeInYears.name, it.toInt()) }
+        pid.ageBirthYear?.let { sdClaim(Attributes.AgeBirthYear.name, it.value.toString()) }
+        pid.familyNameBirth?.let { sdClaim(OidcAssuranceBirthFamilyName.name, it.value) }
+        pid.givenNameBirth?.let { sdClaim(OidcAssuranceBirthGivenName.name, it.value) }
 
         pid.oidcAssurancePlaceOfBirth()?.let { placeOfBirth ->
-            structured(OidcAssurancePlaceOfBirth.NAME) {
-                placeOfBirth.locality?.let { sd("locality", it) }
-                placeOfBirth.region?.let { sd("region", it) }
-                placeOfBirth.country?.let { sd("country", it) }
+            objClaim(OidcAssurancePlaceOfBirth.NAME) {
```

Comment:
This could be also `sdObjClaim` (the same goes for the other composite attributes).